just-install.json

```diff
@@ -2464,6 +2464,51 @@
       },
       "version": "latest-2017"
     },
+    "visual-studio-community": {
```

Comment:
AFAIK having two entries with the same name isn't supported. @lvillani Will this work?

Reply:
Nope, it will not.

Reply:
Should I remove the 2017 version then?
Or is there a way to handle this?

Reply:
Yes. Please remove the 2017 version.

Reply:
Done.